src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Aes.cs

```diff
@@ -11,10 +11,10 @@ namespace System.Runtime.Intrinsics.X86
     /// This class provides access to Intel AES hardware instructions via intrinsics
     /// </summary>
     [CLSCompliant(false)]
-    public static class Aes 
+    public abstract class Aes : Sse2
```

Comment:
Should we make `private` or `internal` constructors on all of them as well?  That way no one tries creating the class, or a class inheriting from this class.

Reply:
Fixed.